GetandTake/Configuration/CustomExceptionHandler.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.AspNetCore.Diagnostics;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace GetandTake.Configuration;
+
+public static class CustomExceptionHandler
+{
+    public static void ExceptionHandler(this WebApplication app)
+    {
+        app.UseExceptionHandler(exceptionHandlerApp =>
+        {
+            exceptionHandlerApp.Run(async context =>
+            {
+                context.Response.StatusCode = StatusCodes.Status500InternalServerError;
+                context.Response.ContentType = Text.Plain;
+
+                await context.Response.WriteAsync("An exception was thrown.");
+
+                var exceptionHandlerPathFeature =
+                    context.Features.Get<IExceptionHandlerPathFeature>();
+
+                if (exceptionHandlerPathFeature?.Error is FileNotFoundException)
```

Comment:
Let's add check for `exceptionHandlerPathFeature == null` before to avoid constant usage of null check, like:
```suggestion
                if (exceptionHandlerPathFeature == null)
                {
                       return;
                }
```